### PR TITLE
Added a flag that excludes Elementary's models.

### PR DESCRIPTION
**Pull request by elongl**

### CODE REVIEW

**Comment by elongl** [comment]
[This](https://github.com/elementary-data/dbt-data-reliability/pull/114) is a requirement.

**Comment by Maayan-s** [comment]
Don't you think it should be true by default?

**Comment by elongl** [comment]
I think that most users would want to be able to look at our own lineage when trying to figure out which tables contain information they might be looking for and for understanding how we generally work.

File: elementary/monitor/cli.py [hunk 1]
```diff
@@ -213,6 +213,12 @@ def monitor(
     default=True,
     help="Whether to open the report in the browser.",
 )
+@click.option(
+    "--exclude-elementary-models",
+    type=bool,
+    default=False,
+    help="Exclude Elementary's own models in the report.",
```
Comment (oravi) [inline elementary/monitor/cli.py:220]: cosmetics - "Exclude Elementary's internal models from the report"

**Pull request merged**